packages/js/github-actions/actions/coverage-report/README.md

```diff
@@ -0,0 +1,39 @@
+# Prepare PHP in GitHub Actions
```

Comment:
Thanks, yes I forgot to fix that title. Although I think we should leave out the PHP, because it should be reusable for JS unit tests. The only part that refers to PHP is the example as it shows how to run it with PHP unit tests.